src/nupic/research/fast_temporal_memory.py

```diff
@@ -175,7 +175,7 @@ def columnForCell(self, cell):
     """
     self._validateCell(cell)
 
-    return int(cell.idx / self.cellsPerColumn)
+    return int(cell / self.cellsPerColumn)
```

Comment:
I suppose though the question is whether this function is part of the API or is only meant to be used internally. It is used externally in various places, but if it shouldn't be, we can create a new accessor for that purpose and leave this one as is (updating the docstring to reflect the actual functionality).